external/wakaama/examples/client/lwm2mclient.c

```diff
@@ -1259,21 +1267,25 @@ int lwm2m_client_main(int argc, char *argv[])
     if (NULL == objArray[8])
     {
         fprintf(stderr, "Failed to create Access Control object\r\n");
+		close(data.sock);
         return -1;
     }
     else if (acc_ctrl_obj_add_inst(objArray[8], instId, 3, 0, serverId)==false)
     {
         fprintf(stderr, "Failed to create Access Control object instance\r\n");
+		close(data.sock);
         return -1;
     }
     else if (acc_ctrl_oi_add_ac_val(objArray[8], instId, 0, 0b000000000001111)==false)
     {
         fprintf(stderr, "Failed to create Access Control ACL default resource\r\n");
+		close(data.sock);
         return -1;
     }
     else if (acc_ctrl_oi_add_ac_val(objArray[8], instId, 999, 0b000000000000001)==false)
     {
         fprintf(stderr, "Failed to create Access Control ACL resource for serverId: 999\r\n");
+		close(data.sock);
         return -1;
```

Comment:
Please, check final source codes for external/wakaama/examples/client/lwm2mclient.c. Do not have these codes. We have changed back wakama to it's original state. I mean original opensource.